lib/BackgroundJobs/IndexerJob.php

```diff
@@ -100,6 +100,7 @@ public function run($argument): void {
 			return;
 		}
 
+		$this->diagnosticService->sendJobStart(static::class, $this->getId());
```

Comment:
would be better to place it at the top so we don't miss any rejected jobs
we would know about the running status with the `sendHeartbeat`